aws/resource_aws_api_gateway_method_settings.go

```diff
@@ -266,3 +271,18 @@ func resourceAwsApiGatewayMethodSettingsDelete(d *schema.ResourceData, meta inte
 
 	return nil
 }
+
+func resourceAwsApiGatewayMethodSettingsImport(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+	idParts := strings.Split(d.Id(), ":")
```

Comment:
I'd recommend using `/` as the separator with `strings.SplitN()` to match other API Gateway resources 👍